pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -96,16 +96,21 @@
 import net.sourceforge.pmd.lang.ast.Node;
 import net.sourceforge.pmd.lang.rule.AbstractRule;
 import net.sourceforge.pmd.lang.rule.ImmutableLanguage;
+import net.sourceforge.pmd.lang.rule.properties.IntegerProperty;
 import net.sourceforge.pmd.renderers.CodeClimateRule;
 
 public abstract class AbstractApexRule extends AbstractRule
         implements ApexParserVisitor, ImmutableLanguage, CodeClimateRule {
 
+    public static final IntegerProperty CODACY_MINUTES_TO_FIX = new IntegerProperty("codacy_minutes_to_fix",
```

Comment:
the codacy properties are unrelated to the object of this PR, please revert this changeset